### DISCUSSION TITLE
Traditional fonts are incorrect and have persisted for many versions.

### DISCUSSION THREAD

**Discussion by faydanube** [Laravel-Lang/common#200] [General]
I've been using laravel-lang/common for zh_TW, and some of the traditional characters have been incorrectly displayed as unsimplified characters for many versions, and I've been manually replacing them every time. Although I'd like to formally submit a pull request, it seems to be a bit difficult for me as a newbie, which has led to several years of delays, so I'm posting here to try to see if anyone can help with this problem, or delay it for a few more years until I know how to submit a pull request.

| search | replace |
| ------ | ------- |
| 当     | 當      |
| 为     | 為      |
| 时     | 時      |
| 信息   | 訊息    |
| 存儲   | 儲存    |
| 帶寬   | 頻寬    |
| 網絡   | 網路    |

**Comment by andrey-helldar** [comment]
@faydanube, we don't have a person assigned to localization of zh_TW in our team yet and we can't discuss the subtleties of Chinese language translation as we are not native speakers.

But, if I understand correctly, you propose to replace the simplified translation with a traditional one?

**Reply by faydanube** [reply]
What I meant to say is that it is incorrect to insert a lot of simplified Chinese characters in the traditional translation.

**Reply by faydanube** [reply]
step by step:

.env

```conf
APP_LOCALE=zh_TW
```

```bash
composer require --dev laravel-lang/common
php artisan lang:update
```

lang/zh_TW/validation.php

<img width="869" alt="image" src="https://github.com/user-attachments/assets/88de61b8-3790-4eaf-a1ce-78f02ce5489b">

```php
    'prohibited_if'        => '当 :other 为 :value 时，:attribute字段被禁止。',
```

"当" is a simplified Chinese character, "當" is a traditional Chinese character.

Referring to the search replace table I gave, I'm sure there are other errors and omissions besides these, except that these are more common, and they have lasted for very many editions. Perhaps if it were easier for non-experienced developers to submit pull requests, this problem would be greatly improved.

**Reply by andrey-helldar** [reply]
I got it. The phrases have a mixed translation. Unfortunately, this is a consequence of Google Translate.

You can submit a PR with changes, you need to fork the repository, make changes to the localization files and create a PR.

There is also a guide on our site: https://laravel-lang.com/contributions.html
